Pratiksha/Javascript/Assignment2/Q2.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Grading</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <form>
+      Subj1:
+      <input type="text" id="val1" /><br /><br />
+      Subj2:
+      <input type="text" id="val2" /><br /><br />
+      Subj3:
+      <input type="text" id="val3" /><br /><br />
+      Subj4:
+      <input type="text" id="val4" /><br /><br />
+      Subj5:
+      <input type="text" id="val5" /><br /><br />
+      <button
+        onclick="calculate()"
+        type="button"
+        class="ml-4 btn btn-primary center"
+        id="cal"
+      >
+        Calculate
+      </button>
+      <button
+        onclick="grade()"
+        type="button"
+        class="ml-4 btn btn-primary center"
+      >
+        Grade
+      </button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      var c;
+      function calculate() {
+        var x, y, z, a, b;
```

Comment:
change variable names in every file don't use a, b, c as a variable name, variable name should be related to functionality